### PR TITLE
Metadata Migrations script

### PR DESCRIPTION
**Pull request by lkostrowski**
todo

- [x] up/down (migrate / restore to original)
- [x] test

### CODE REVIEW

File: apps/invoices/scripts/migrations/v1-to-v2/run-migration.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  const allEnvs = await fetchCloudAplEnvs();
+
+  const results = await Promise.all(
+    allEnvs.map((env) => {
+      const metadataManager = getMetadataManagerForEnv(env.saleorApiUrl, env.token);
+
+      return Promise.all([
+        metadataManager.get("app-config", env.saleorApiUrl),
+        metadataManager.get("app-config-v2"),
```
Comment (peelar) [inline apps/invoices/scripts/migrations/v1-to-v2/run-migration.ts:27]: I suggest exporting these "app-config" and "app-config-v2" tokens from somewhere.

File: apps/invoices/scripts/migrations/migration-utils.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  } catch (e) {
+    return null;
+  }
+};
+
+export const verifyRequireEnvs = () => {
+  const requiresEnvs = ["SALEOR_CLOUD_TOKEN", "SALEOR_CLOUD_RESOURCE_URL", "SECRET_KEY"];
+
+  if (!requiresEnvs.every((env) => process.env[env])) {
+    throw new Error(`Missing envs: ${requiresEnvs.join(" | ")}`);
```
Comment (peelar) [inline apps/invoices/scripts/migrations/migration-utils.ts:29]: Nitpick:

`verifyRequireEnvs` -> `verifyRequiredEnvs`
`requiresEnvs` -> `requiredEnvs`

File: .changeset/real-onions-move.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-invoices": patch
+---
+
+Update app-sdk to 0.38.0
```
Comment (peelar) [inline .changeset/real-onions-move.md:5]: Missing notion of the addition of metadata migration scripts

Comment (lkostrowski) [reply .changeset/real-onions-move.md:5]: Migration fact is not changelog, actually its the opposite - similar to refactor its purpose is to be *not* noticeable, here is some short cheatsheet on changelogs

https://keepachangelog.com/en/1.0.0/

**Review by peelar — APPROVED** [review]

**Pull request merged**